assets/dev/js/editor/elements/views/base.js

```diff
@@ -341,7 +341,23 @@ BaseElementView = BaseContainer.extend( {
 	},
 
 	attachElContent( html ) {
-		this.$el.empty().append( this.getHandlesOverlay(), html );
+		if ( elementor.previewView.isBuffering || ! elementorCommon.config.experimentalFeatures.e_nested_atomic_repeaters || 'nested-accordion' !== this?.model?.config?.name ) {
```

Comment:
doing an action and returning is not the best practice 
maybe extract the actions to separate functions

 like :

if ( ShouldDoSomethig() ) {
DoSomething()
}else{
DoSomethingElse()
}